File: Programacao  2/C/Ex20.c
#include <stdio.h>
 
int main(int argc, char *argv[]){
  float a, b,M;
  
  printf("\n Digite a nota da P1: ");
  scanf("%f", &a);

  printf("Digite a nota da P2: ");
  scanf("%f", &b);

  

  M= (a + (2*b)) / 3;

  if (M>=5){

      printf("Aluno Aprovado");
  }

else{

  printf( " Aluno Reprovado");
}

return 0;

 
}

File: Programacao  2/Phyton/Ex20.py

P1 = float(input('Digite a nota da p1: '))
P2 = float(input('Digite a nota da p2: '))

Media = (P1 + P2*2)/3
print(f"A Nota Final é:{Media: .2f}")

if Media>=5:
  print(f"Aprovado!")
else:
  print(f"Reprovado!")